pkgs/servers/owncast/default.nix

```
@@ -1,43 +1,34 @@
-{ lib, buildGoModule, fetchFromGitHub, nixosTests, bash, which, ffmpeg, makeWrapper, coreutils, ... }:
-
-buildGoModule rec {
+{
+  lib
+  ,buildGoModule
+  ,callPackage
```

This formatting is weird, standard formatting would be like

```suggestion
{ lib
, buildGoModule
, callPackage
```